src/puppetlabs/puppetdb/admin.clj

```diff
@@ -19,11 +18,9 @@
   (cmdi/context "/v1/archive"
                 (cmdi/wrap-routes
                  (cmdi/POST "" request
-                            (let [{{:strs [archive command_versions]} :multipart-params} request]
-                              (import/import! (:tempfile archive)
-                                              (json/parse-string command_versions true)
-                                              submit-command-fn)
-                              (http/json-response {:ok true})))
+                            (import/import! (get-in request [:multipart-params "archive" :tempfile])
```

Comment:
Hey it worked last time. Anyway we seem to need it in this case because libcurl.